internal/cmdrender/executor.go

```diff
@@ -316,8 +316,33 @@ func fnChain(pl *kptfilev1alpha2.Pipeline, pkgPath types.UniquePath) ([]kio.Filt
 	return runners, nil
 }
 
+// errorIfDuplicateAnnotation returns an error if the same index/path is on multiple resources
+func errorIfDuplicateAnnotation(nodes []*yaml.RNode) error {
```

Comment:
We don't need to include "error" in the method name since it returns an error. Alternative name could be: `checkDuplicatePath`, `detectPathConflicts`